package.json

```diff
@@ -43,5 +43,17 @@
   },
   "engines": {
     "node": ">= 0.10"
+  },
+  "keywords": [
+    "hypernova",
+    "client"
+  ],
+  "directories": {
```

Comment:
i found https://docs.npmjs.com/files/package.json#directories , but this field isn't used for anything, so i'd rather omit it.